axon-framework/events/event-processors/README.md

```diff
@@ -307,12 +307,12 @@ A `JpaSequencedDeadLetterQueue` configuration example:
 ```java
 public class DeadLetterQueueExampleConfig {
     
-    public ConfigurerModule configureDeadLetterQueueFor(String processingGroup) {
+    public ConfigurerModule configureDeadLetterQueueFor(EventProcessingConfigurer configurer) {
         return configurer -> configurer.eventProcessing(
             eventProcessingConfigurer -> eventProcessingConfigurer.registerDeadLetterQueue(
-                processingGroup,
+                "MyProcessingGroup",
                 configuration -> JpaSequencedDeadLetterQueue.builder()
-                                                            .processingGroup(processingGroup)
+                                                            .processingGroup("MyProcessingGroup")
```

Comment:
Nit: the other code sample where we configure something for a Processing Group (the `ListenerInvocationErrorHandler`) uses `"my-processing-group"` instead of `MyProcessingGroup`. 
It would be good if these samples stay consistent with one another.